src/main/java/org/jboss/xavier/integrations/migrationanalytics/business/versioning/ManifestPathVersion.java

```diff
@@ -0,0 +1,22 @@
+package org.jboss.xavier.integrations.migrationanalytics.business.versioning;
+
+import lombok.Data;
+
+@Data
+public class ManifestPathVersion implements Comparable<ManifestPathVersion> {
+    private final String version;
+
+    @Override
+    public int compareTo(ManifestPathVersion versionB) {
+        int major = Integer.valueOf(version.split("_")[0]).compareTo(Integer.valueOf(versionB.getVersion().split("_")[0]));
```

Comment:
No need to execute the same `version.split("_")` and `versionB.getVersion().split("_")` splits every time we need to get the n-th element.
Please do the splits at the beginning and then reference the `String[]` you'll have.

Reply:
I will check that

Reply:
ok